API/Controllers/ElectricityController.cs

```diff
@@ -0,0 +1,69 @@
+﻿using API.Data;
+using API.Models;
+using Microsoft.AspNetCore.Http;
+using Microsoft.AspNetCore.Mvc;
+using Microsoft.EntityFrameworkCore;
+
+namespace trackingapi.Controllers
+{
+    [Route("api/[controller]")]
+    [ApiController]
+    public class ElectricityController : ControllerBase
+    {
+        private readonly IssueDbContext _context;       //Allow controller to access the database
+
+        public ElectricityController(IssueDbContext context) => _context = context;
+
```

Comment:
Why so many empty lines?

Reply:
Empty lines were created for convenience, forgot to delete them